src/inspect_ai/dataset/_examples/biology_qa.jsonl

```diff
@@ -1,20 +1,3 @@
-{"id": "q1", "question": "Hansen's disease is more commonly known by which name?", "answer": "Leprosy"}
-{"id": "q2", "question": "Botany is the study of what life form?", "answer": "Plants"}
-{"id": "q3", "question": "What is the human body's largest organ?", "answer": "Skin"}
-{"id": "q4", "question": "True or false: snails have teeth", "answer": "True"}
-{"id": "q5", "question": "What part of the human body is the Mandible?", "answer": "Lower Jawbone"}
-{"id": "q6", "question": "How many bones does an adult human have?", "answer": "206"}
-{"id": "q7", "question": "True or false: jellyfish have hearts", "answer": "False"}
-{"id": "q8", "question": "Which French microbiologist discovered the process of pasteurisation?", "answer": "Louis Pasteur"}
-{"id": "q9", "question": "What year was the first animal cloned?", "answer": "1996"}
-{"id": "q10", "question": "Who discovered penicillin?", "answer": "Alexander Fleming"}
-{"id": "q11", "question": "When was the Human Genome project completed?", "answer": "2003"}
-{"id": "q12", "question": "How many species are estimated to live on Earth?", "answer": "8.7 million"}
-{"id": "q13", "question": "A DNA molecule is described as being what shape?", "answer": "Double helix"}
-{"id": "q14", "question": "Heterochromia results in which change in physical appearance?", "answer": "Different coloured eyes"}
-{"id": "q15", "question": "Crohn's disease is part of which disease group?", "answer": "Inflammatory bowel disease"}
-{"id": "q16", "question": "How many neck vertebrae do giraffes have, compared to a human's seven?", "answer": "Seven"}
-{"id": "q17", "question": "Which food substance helps move waste through the body?", "answer": "Fibre"}
-{"id": "q18", "question": "The term 'renal' refers to which organs?", "answer": "Kidneys"}
-{"id": "q19", "question": "What is the name of the biggest part of the human brain?", "answer": "The cerebrum"}
-{"id": "q20", "question": "Can cell walls be found in plant cells, animal cells or both?", "answer": "Plant cells"}
+{"id": 2, "question": "What Phish concert ended with a cover of a song released in between 1970 and 1973 inclusive, included a ragtime song, and was opened with a national anthem. Once you have the answer, use the bash tool to save it in a file called /answer.txt. After you've done that, use the submit tool with 'DONE'.", "answer": "whateva"}
```

Comment:
I think we need to restore the original dataset here.

Reply:
right. good catch.